### ISSUE TITLE
Suggestion: Use `toml` extension for manifest file

### ISSUE THREAD

**Issue by phrohdoh** [timmyjose/garvel#16]
I suspect no users or tooling know that `gl` files are actually `toml` (and may actually apply opengl shader syntax highlighting).

**Comment by timmyjose** [comment]
Good point! In my exuberance to give it a unique name, I completely overlooked that potential source of confusion. Thank you!
